content/tutorials/manage-the-rippled-server/configuration/configure-full-history.md

```diff
@@ -0,0 +1,98 @@
+# Configure Full History
+
+In its default configuration, the `rippled` server automatically deletes outdated history of XRP Ledger state and transactions as new ledger versions become available. This is sufficient for most servers, which do not need older history to know the current state and process transactions. However, it can be useful for the network if some servers provide as much history of the XRP Ledger as possible.
+
+## Warnings
+
+Storing full history is expensive. As of 2018-12-11, the full history of the XRP Ledger occupies approximately **9 terabytes** of disk space, which must be entirely stored on fast solid state disk drives for proper server performance. Such a large amount of solid state storage is not cheap, and the total amount of history you must store increases by approximately 12 GB per day.
+
+Acquiring full history from the peer-to-peer network takes a long time (several months) and requires that your server has sufficient system and network resources to acquire older history while keeping up with new ledger progress. To get a faster start on acquiring ledger history, you may want to find another server operator who has a large amount of history already downloaded, who can give you a database dump or at least allow your server to explicitly peer with theirs for a long time to acquire history. The server can load ledger history from a file and verify the integrity of the historical ledgers it imports.
+
+You do not need a full history server to participate in the network, validate transactions, or know the current state of the network. Full history is only useful for knowing the outcome of transactions that occurred in the past, or the state of the ledger at a given time in the past. To get such information, you must rely on other servers having the history you need.
+
+If you want to contribute to storing the history of the XRP Ledger network without storing the full history, you can [configure history sharding](configure-history-sharding.html) to store randomly-selected chunks of ledger history instead.
+
+## Configuration Steps
+
+To configure your server to acquire and store full history, complete the following steps:
+
+1. Stop the `rippled` server if it is running.
+
+        $ sudo systemctl stop rippled
+
+0. Remove (or comment out) the `online_delete` and `advisory_delete` settings from the `[node_db]` stanza of your server's config file:
+
+        [node_db]
+      	type=RocksDB
+      	path=/var/lib/rippled/db/rocksdb
+      	open_files=2000
+      	filter_bits=12
+      	cache_mb=256
+      	file_size_mb=8
+      	file_size_mult=2
+        compression=1
+      	#online_delete=2000
+      	#advisory_delete=0
+
+    You can store full history with either RocksDB or NuDB as the key-value store behind the ledger store. NuDB requires less RAM but RocksDB requires less disk space. For more information, see [Capacity Planning](capacity-planning.html).
```

Comment:
Don't store full history with rocksdb. Just nudb. Based on RAM utilization of each. RAM requirements of rocksdb for full history are unknown.